app/views/components/docs/actions-group.yml

```diff
@@ -0,0 +1,20 @@
+name: Actions group
```

Comment:
@alex-ju do we need these example docs?

Reply:
We use them to render the component preview in `/component-guide`

Reply:
Ah cool, thanks for clarifying 👍